tests/verify_membackend.erl

```diff
@@ -152,7 +152,10 @@ check_put_delete(Node) ->
     Mem = get_used_space(Pid, Node),
 
     %% this is meh, but the value isn't always the same length.
-    case (Mem == MemBaseline - 1142) orelse 
+    %% It seems to be the size of a Riak Object
+    case (Mem == MemBaseline - 1146) orelse
```

Comment:
It makes sure that if a delete happens, memory is reclaimed.  So one thing might be to grab the riak object using the Key and ask the system what size it is and then make sure when its deleted, the memory drops by that amount.